src/view-data/method.ts

```diff
@@ -28,8 +28,7 @@ export interface Method {
   readonly externalDocs: string;
   readonly parameters: TypeSpecParameter[];
   readonly headers: Header[];
-  readonly successfulResponseType: string;
-  readonly successfulResponseTypeIsRef: boolean;
+  readonly responseTypes: string;
```

Comment:
Breaking API change.

Reply:
Thinking a bit about this - Let's try to keep it backwards compatible. We can keep the old ones and mark them as deprecated, and remove them at a later point in time